test/external/process_replay/process_replay.py

```diff
@@ -11,16 +12,18 @@
 for offset in tqdm(range(0, row_count, page_size)):
   cur.execute(f"SELECT val FROM 'process_replay_{VERSION}' LIMIT ? OFFSET ?", (page_size, offset))
   for row in cur.fetchall():
-    compare_k, compare_src = pickle.loads(row[0])
+    compare_k, compare_src, time_baseline = pickle.loads(row[0])
     k = Linearizer(*compare_k.ast, opts=compare_k.opts)
     for opt in compare_k.applied_opts: k.apply_opt(opt)
     good_src = k.opts.render(to_function_name(compare_k.name), k.linearize().uops)
-    try: assert compare_src == good_src
+    time_ = timeit(UOpGraph(k.uops).linearize)
```

Comment:
hmm This is wrong actually, k.uops is the entire linearized version
Why don't you just time k.linearize().uops?